api/db-cdr/generate-cdr/cb-criteria-refactor/debug/comp-tables.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
```

Comment:
Adding here since this is the first file in this directory...
I should have noticed this sooner, but could we rename this directory to:
`api/db-cdr/generate-cdr/cb-criteria-parallel` or something more descriptive in the long term than `refactor`?

Reply:
done